src/lib/components/inputs/Input.svelte

```diff
@@ -0,0 +1,99 @@
+<script lang="ts">
+  export let value: string = "";
```

Comment:
Do not use explicit types @Astitva877 . 

For example, 
export let isRequired: boolean = false; (no need for boolean)
export let inputId: string = ""; (no need for string)

Reply:
Resolved.